pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -40,6 +40,10 @@ func (c *Cluster) makeDeployment(mgrConfig *mgrConfig) *apps.Deployment {
 		ObjectMeta: metav1.ObjectMeta{
 			Name:   mgrConfig.ResourceName,
 			Labels: c.getPodLabels(mgrConfig.DaemonID),
+			Annotations: map[string]string{
+				"prometheus.io/scrape": "true",
+				"prometheus.io/port":   strconv.Itoa(metricsPort),
```

Comment:
The annotations are already set on line 102-109 on the `podSpec` (the code you want to remove).